autofill/frame-form-submission-child.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Form submission detection and autofill within an iframe</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <p id="demo"></p>
+
+  <div class="">
+    <p>Submitting this form should prompt to save credentials.</p>
+
+    <form id="click-form" novalidate action="/autofill/form-submission.html">
+      <h3>Sign in form with click handler</h3>
+      <fieldset>
+        <label for="email">Email</label>
+        <input id="email" type="email">
+        <label for="password">Password</label>
+        <input id="password" type="password">
+        <button type="submit">Sign in</button>
+      </fieldset>
+    </form>
+    <script>
+      const clickForm = document.getElementById('click-form');
+      const clickFormButton = clickForm.querySelector('button');
+      clickFormButton?.addEventListener('click', (e) => {
+          e.preventDefault();
+          e.stopImmediatePropagation();
+          // Note that form.submit() does not fire a submit event
+          clickForm.submit();
```

Comment:
I suggest also adding a visual indication that the form was indeed submitted. See [this](https://github.com/duckduckgo/privacy-test-pages/pull/89/files#diff-864a4e1a3d54def77f5ee9e7507d27bc22cf8fe3472ae053d8ce6184b3f78059R46) as an example.